Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue1763.cs

```diff
@@ -6,21 +6,40 @@
 using System.Runtime.CompilerServices;
 using System.Text;
 using System.Threading.Tasks;
-
 using Xamarin.Forms.CustomAttributes;
 using Xamarin.Forms.Internals;
 
+#if UITEST
+using NUnit.Framework;
+#endif
+
 namespace Xamarin.Forms.Controls.Issues
 {
 	[Preserve (AllMembers=true)]
 	[Issue (IssueTracker.Github, 1763, "First item of grouped ListView not firing .ItemTapped", PlatformAffected.WinPhone, NavigationBehavior.PushAsync)]
-	public class Issue1763 : TabbedPage
+	public class Issue1763 : TestTabbedPage
```

Comment:
Looks like you might have accidentally pulled in your changes from https://github.com/xamarin/Xamarin.Forms/pull/5907.

Reply:
removed commit from PR